packages/genesys-spark-components/package.json

```diff
@@ -66,7 +66,7 @@
     "@floating-ui/dom": "1.7.0",
     "@js-temporal/polyfill": "0.5.1",
     "@vvo/tzdb": "6.161.0",
-    "google-libphonenumber": "3.2.40",
+    "google-libphonenumber": "^3.2.41",
```

Comment:
I know it's only a patch update, but I'm a bit wary of any libphonenumber updates. I'd like to check the gux-phone-input in the demo link, but it seems to be taking a while to deploy.
Will check the npm libraries to see the difference too